src/main/kotlin/racing_car/domain/car/RandomMovingEngine.kt

```diff
@@ -0,0 +1,17 @@
+package racing_car.domain.car
+
+class RandomMovingEngine(
+    private val minRandomValue: Int = 0,
+    private val maxRandomValue: Int = 9,
+    private val moveThreshold: Int = 4,
+    private val defaultMoveDistance: Int = 1
+) :
+
+    Engine {
+    override fun getMoveDistance(): Int {
+        if ((minRandomValue..maxRandomValue).random() >= moveThreshold) {
+            return defaultMoveDistance
+        }
+        return 0
+    }
```

Comment:
이 함수를 호출할 때마다 `(minRandomValue..maxRandomValue)`가 생성될 것 같아요.
별도의 프로퍼티로 관리해주면 어떨까요?
```suggestion
   private val availableRange: IntRange = (min..max)
   
   override fun getMoveDistance(): Int {
         if (availableRange.random() >= moveThreshold) {
             return defaultMoveDistance
         }
         return 0
     }
```

Reply:
(minRandomValue..maxRandomValue) 가 하나의 객체군요!